src/json_mst.rs

```diff
@@ -0,0 +1,148 @@
+use num_bigint::BigUint;
```

Comment:
Can you provide some in-line comment to `json_mst.rs` structs and functions too?